Framework/Algorithms/src/PolarizationCorrectionWildes.cpp

```diff
@@ -333,10 +326,14 @@ void PolarizationCorrectionWildes::init() {
       std::make_unique<API::WorkspaceProperty<API::WorkspaceGroup>>(Prop::OUTPUT_WS, "", Kernel::Direction::Output),
       "A group of polarization efficiency corrected workspaces.");
 
-  const auto spinStateValidator = std::make_shared<SpinStateValidator>(std::unordered_set<int>{1, 2, 3, 4}, true);
+  const auto flipperConfigValidator = std::make_shared<SpinStateValidator>(std::unordered_set<int>{1, 2, 3, 4}, true);
   declareProperty(Prop::FLIPPERS,
-                  Flippers::OffOff + ", " + Flippers::OffOn + ", " + Flippers::OnOff + ", " + Flippers::OnOn,
-                  spinStateValidator, "Flipper configurations of the input workspaces.");
+                  std::string(FlipperConfigurations::OFF_OFF) + ", " + FlipperConfigurations::OFF_ON + ", " +
+                      FlipperConfigurations::ON_OFF + ", " + FlipperConfigurations::ON_ON,
+                  flipperConfigValidator, "Flipper configurations of the input workspaces.");
+  const auto spinStateValidator =
+      std::make_shared<SpinStateValidator>(std::unordered_set<int>{0, 1, 2, 4}, true, '+', '-', true);
```

Comment:
I was just wondering why the allowed number of spins was set to `{0, 1, 2, 4}` here when it looks like it's set to `{0, 2, 4}` in `PolarizationEfficiencyCor`?

Reply:
Ah, good spot. I'd misunderstood the possible outputs when I implemented the changes in Wildes, but had worked them out when I got to EfficiencyCor.

Reply:
I realised reading through the PR that it's more complex than it first appears, so great job spotting that and figuring it all out in the end!